### PR TITLE
Make preview map ignore mouse events unless clicked

### PR DESCRIPTION
**Pull request by steve9164**
### What this PR does
See #1182.
- Update Magda's TerriaMap to display the standard zoom controls and remove the `AugmentedVirtuality` chunk that was never used
- Update Magda web UI to:
    + Fix 6 pixel jump when TerriaMap finishes loading
    + Correct "Open/View in NationalMap" buttons to use NationalMap's proper brand name
    + Stop scrolling from zooming the preview map until the user clicks on the map
### Checklist

-   [x] Unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column
-   [x] There are sufficient comments for my code to be understandable - and I realise reviewers will pull me up on it if not!

### CODE REVIEW

File: magda-web-client/src/UI/DataPreviewMap.js [hunk 1]
```diff
@@ -23,80 +24,85 @@ export const isSupportedFormat = function(format) {
     );
 };
 
+// React 16.3 advice for replacing prop -> state updates for computations
+const determineDistribution = memoize(function determineDistribution(
```
[the remark this answers is not in the document]
Comment (steve9164) [reply magda-web-client/src/UI/DataPreviewMap.js]: I have fixed some of the problems with that function

**Comment by AlexGilleran** [comment]
@steve9164 are you still going to add the + / - buttons to this?

**Comment by steve9164** [comment]
It has zoom buttons. It's in magda-preview-map so the UI only preview doesn't include it.

**Comment by AlexGilleran** [comment]
So how do we include it in the preview?

**Pull request merged**